engines/wintermute/ad/ad_entity.cpp

```diff
@@ -1105,6 +1176,15 @@ bool AdEntity::persist(BasePersistenceManager *persistMgr) {
 
 	persistMgr->transferPtr(TMEMBER_PTR(_theora));
 
+#ifdef ENABLE_FOXTAIL
```

Comment:
Hrm, might as well consider making the persistMgr Foxtail-aware, and just only persisting these if we are FoxTail.

Reply:
Fixed at 2be4dde404c39b2bdc37bb74d8095084f95bb7e4.

This was the only place shere PersistentManager related changes were made for FoxTail, so now  this pull request finally should keep savegame compatibility!